src/components/Headline/StyledHeadline1.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+import {pxToRem} from '../../utils/unit';
+
+const StyledH1 = styled.h1`
+	color: var(--turq);
+	font-size: ${pxToRem(20)};
+`;
+
+export default function StyledHeadline1() {
+	return (
+		<>
+			<StyledH1>Great to have you here</StyledH1>
+		</>
```

Comment:
As you are only returning one element you do not need the fragment here.

```suggestion
			<StyledH1>Great to have you here</StyledH1>
```